cloudinit/net/activators.py

```diff
@@ -184,6 +184,30 @@ def bring_down_interface(device_name: str) -> bool:
         cmd = ["nmcli", "device", "disconnect", device_name]
         return _alter_interface(cmd, device_name)
 
+    @classmethod
+    def bring_up_interfaces(cls, device_names: Iterable[str]) -> bool:
```

Comment:
Where does this get called from? Given that this is a new function I assume there are no existing calls to this function?

Reply:
There's a base class impl here: https://github.com/canonical/cloud-init/blob/main/cloudinit/net/activators.py#L74
so this will override it for network manager. 

It gets called from https://github.com/canonical/cloud-init/blob/main/cloudinit/distros/__init__.py#L466